src/Tasks/Common/Resources/Strings.resx

```diff
@@ -237,6 +237,10 @@
     <value>NETSDK1032: The RuntimeIdentifier platform '{0}' and the PlatformTarget '{1}' must be compatible.</value>
     <comment>{StrBegin="NETSDK1032: "}</comment>
   </data>
+  <data name="PropertyValueShouldBeLowercase" xml:space="preserve">
+    <value>NETSDK1033: The value of '{0}' is expected to be in lower-case but the actual casing is '{1}'. This value might be used as a part of a path which could lead into unexpected file not found errors (i.e. in case-sensitive file-systems). So, please use lower-case for '{0}' value.</value>
```

Comment:
The main motivation is, when a team of devs do the following… Then, for the guy (**ME**) who stuck at figuring out what's wrong with the build, this message would've been helpful to debug and fix the build, instead of banging my head against the wall, in a race against time!

Let's say, we could set TF and RID programmatically…

```proj
  <PropertyGroup>
    <TargetFramework Condition="'$(TargetFramework)' == ''">net3.1</TargetFramework>
    <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">win-x64</RuntimeIdentifier>
  </PropertyGroup>
```

And a build script that does the following…

```ps1

  # 'tfx' and 'rid' could have uppercase chars
  tfx = get_tfx() # get framework moniker
  rid = get_rid() # get runtime moniker

  msbuild -t:Build;Validate -p:TargetFramework=$tfx -p:RuntimeIdentifier=$rid

  path = make_path($tfx, $rid) # contruct path using 'tfx' and 'rid'

  # some logic to copy files or run tests against those files…
```

Then, the above script would fail, since, the file is not found in a case-sensitive file system.

So, to me, this is not noise, this is godsend!